independent-publisher-connectors/myStrom/apiDefinition.swagger.json

```diff
@@ -0,0 +1,640 @@
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "",
+                  "title": "Code"
+                },
+                "message": {
+                  "type": "string",
+                  "description": "",
+                  "title": "Message"
+                },
+                "color": {
+                  "type": "string",
+                  "description": "Current color in HSV format.",
+                  "title": "Color"
+                }
+              }
+            }
+          },
+          "default": {
+            "description": "Default response",
+            "schema": {}
+          }
+        },
+        "summary": "Toggle device",
+        "description": "Device toggle request is used to change state of a device and set its parameters",
+        "operationId": "ToggleDevice",
+        "parameters": [
+          {
+            "name": "Device ID",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Device ID",
+            "x-ms-summary": "Device ID",
+            "x-ms-visibility": "important",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "color",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The color we set the bulb to. HSV is of form <UINT 0..360>;<UINT 0..100>;<UINT 0..100>",
+            "x-ms-summary": "Color",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "name": "ramp",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "description": "Transition time from the light\u2019s current state to the new state. [ms]",
+            "x-ms-summary": "Ramp",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "name": "value",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "description": "Dim value for devices that support dim action",
+            "x-ms-summary": "Dim Value",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "name": "action",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "default": "on",
+            "description": "Switch action",
+            "x-ms-summary": "Action",
+            "enum": [
+              "on",
+              "off",
+              "toggle",
+              "dim"
+            ]
+          },
+          {
+            "name": "authToken",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "Authentication Token",
+            "x-ms-summary": "Authentication Token",
+            "format": "password",
+            "x-ms-visibility": "important"
+          }
+        ]
+      }
+    },
+    "/device": {},
```

Comment:
@tposzytek Empty operation definations can be removed as they would not add any value to the connector.

Reply:
Done